packages/help/help.py

```diff
@@ -0,0 +1,41 @@
+from halibot import HalModule
+
+class Help(HalModule):
+
+	def init(self):
+		self.commands = {
+			"help": self.help_
+		}
+
+	def receive(self, msg):
+		if not msg.body:
+			return
+		body = msg.body.split(" ")
+		func = self.commands.get(body[0][1:])
+		if func:
+			func(body[1:], msg=msg)
```

Comment:
Should we be checking for a `!` or something here? I presume so, given you chomp the first character in the first word.

Reply:
There's sort of a reason for that -- the `CommandModule` class handles which character to use for a command, so I'm skipping over that a bit. That said, there could be false positives here, so that should probably be cleaned up.

Reply:
Then should you not be inheriting from `CommandModule`? And in that case you do not need to implement `receive` at all?

Reply:
Ah sorry, I meant to say I was leaving this open for `CommandModule` to take care of once that gets upstream. Didn't want to spent a ton of effort writing a command parser when that whole `.receive()` function was going to get deleted anyway.